### PR TITLE
new open link commit

### PR DESCRIPTION
**Pull request by Nethracs**
removed openlink branch, branched out from develop and added the changes.

### CODE REVIEW

**Review by PrashanthKumarM — APPROVED** [review]
Looks good to go

File: actionlist/app/components/action_item_body.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+/* eslint no-underscore-dangle: ["error", { "allow": ["_links"] }] */
```
Comment (burnumd) [inline actionlist/app/components/action_item_body.js]: This eslint directive should appear closer to where it is used (and re-enabled ASAP).

File: actionlist/app/components/action_item_body.js [hunk 2]
```diff
@@ -38,6 +37,7 @@ const styles = StyleSheet.create({
   },
 });
 
+
```
Comment (burnumd) [inline actionlist/app/components/action_item_body.js]: Remove this extra whitespace, please.

**Review by burnumd — CHANGES_REQUESTED** [review]

File: actionlist/app/components/action_item_body.js [hunk 3]
```diff
@@ -98,4 +102,4 @@ ActionItemBody.defaultProps = {
   },
 };
 
-export default ActionItemBody;
+export default ActionItemBody;
```
Comment (burnumd) [inline actionlist/app/components/action_item_body.js]: Add a newline at the end of this file, please.

**Review by burnumd — APPROVED** [review]

**Pull request merged**